ArrayUtils.js

```diff
@@ -1,21 +1,84 @@
 class ArrayUtils{
 
-    isEmpty(array){return false;}
 
-    append(original, value){return original;}
+     isEmpty(array){
+        if (array.length !== 0){
+        return false;
+        }
+    }
+    
+    append(original, value){
```

Comment:
This is an infinite loop, each time you add an element, the length goes up. This will happen each iteration. (-3)